openff/toolkit/_tests/test_toolkits.py

```diff
@@ -4529,6 +4529,50 @@ def test_call_raise_first_error(self):
             )
 
 
+@requires_openeye
+@requires_ambertools
+@requires_rdkit
+class TestToolkitRegistryManager:
+    def test_openeye_registry(self):
+        from openff.toolkit.utils.toolkit_registry import toolkit_registry_manager
+
+        openeye_registry = ToolkitRegistry(toolkit_precedence=[OpenEyeToolkitWrapper])
+
+        with toolkit_registry_manager(openeye_registry):
+            assert "OpenEye" in str(GLOBAL_TOOLKIT_REGISTRY)
+            assert "RDKit" not in str(GLOBAL_TOOLKIT_REGISTRY)
+
+            create_ethanol().assign_partial_charges(
+                partial_charge_method="am1bccelf10",
+            )
+
+    def test_no_openeye_registry(self):
```

Comment:
The best way to test this would probably involve a method that RDKit provides but OpenEye doesn't, but I couldn't think of any. I think the checks here should be enough to cover the bases.